config/config_hmpxv1.yaml

```diff
@@ -17,7 +17,8 @@ auspice_name: "monkeypox_hmpxv1"
 ## filter
 min_date: 2017
 min_length: 10000
-sequences_per_group: 1000
+sequences_per_group: 20
+groupby: "clade lineage year month country"
```

Comment:
Not sure I'm in favour of this as it means discovering new lineages will be harder.

This will downsample mostly B.1's that are not assigned any lineage.

What if we leave out the lineage here? Clade doesn't do anything either so can be deleted as well, leaving us with `year month country`, we could then adjust this to say 50 to reduce the amount of sequences from Germany.

Reply:
Yes, we could reduce this to `year month country`